ai-platform-unified/notebooks/unofficial/featurestore/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1342 @@
+{
```

Comment:
<p>We don't typically need to add this. I understand it's helpful but it's a little non-standard.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>removed</p>